repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/wrappers/response/ResponseWrapperTest.groovy

```diff
@@ -240,6 +269,7 @@ class ResponseWrapperTest extends ReposeValveTest {
         411          | "Static Message"
     }
 
+    @Ignore('RMS is not currently supported')
```

Comment:
I am debating removed these tests entirely. I think the idea was that the only (reliable) way to access the status message was through the RMS (since it had to be access through a field on the response). However, since we are dropping the WAR deployment and only supporting Jetty, we know that the behavior of `sendError` is to set the reason phrase. As such, we can now check the reason phrase directly on the response rather than through the RMS. As such, RMS testing can be independent.

Reply:
makes sense to me